Sources/GRPC/HTTP1ToRawGRPCServerCodec.swift

```diff
@@ -270,17 +338,24 @@ extension HTTP1ToRawGRPCServerCodec: ChannelOutboundHandler {
         //
         // We can try! here because the server does not support compression at the moment and this
         // only throws when compression fails.
-        try! messageWriter.write(messageBytes, into: &self.responseTextBuffer)
-
-        // Since we stored the written data, mark the write promise as successful so that the
-        // ServerStreaming provider continues sending the data.
-        promise?.succeed(())
+        do {
```

Comment:
Is this for gRPC-Web? If so, should we support compression for gRPC-Web at all, given that the recipient probably expects a non-binary response?

Reply:
The gRPC-Web spec doesn't mention disallowing compression. I just checked their repo as well, there's some code in there related to gzip so this shouldn't be an issue.